pkg/cmd/server/start/start_master.go

```diff
@@ -527,6 +516,44 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 	return nil
 }
 
+// Look in two potential places where ClusterResourceOverrideConfig can be specified
+func getResourceOverrideConfig(oc *origin.MasterConfig) (*overrideapi.ClusterResourceOverrideConfig, error) {
+	if overrideConfig, err := tryOverrideConfig(oc.Options.AdmissionConfig); err != nil {
```

Comment:
move the if to its own line, followed by 2 if's ... 
https://golang.org/doc/effective_go.html#if